.circleci/config.yml

```diff
@@ -1,5 +1,7 @@
 version: 2.1
 
+# Cache key bump: 1
```

Comment:
can we remove this now?

Reply:
Yes.  We can also leave it and just change it to `2` next time we need it.  lol.  (I do usually remove it.)